app/assets/javascripts/components/locales/ja.jsx

```diff
@@ -76,6 +76,24 @@ const ja = {
   "notifications.column_settings.show": "カラムに表示",
   "notifications.column_settings.sound": "通知音を再生",
   "notifications.settings": "カラム設定",
+  "onboarding.next": "次へ",
+  "onboarding.page_one.welcome": "Mastodonへようこそ！",
+  "onboarding.page_one.federation": "Mastodonは誰でも参加できるSNSです。",
+  "onboarding.page_one.handle": "あなたは今数あるMastodonインスタンスの1つである{domain}にいます。あなたのフルハンドルは{handle}です。",
```

Comment:
@Gargron It's ok to merge this for the moment. And fix this point after the time.